File: JAVA/Ejercicio4.java
import java.util.Scanner;
public class Ejercicio4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Introduce tres números enteros separados por espacios: ");
        int n1 = sc.nextInt();
        int n2 = sc.nextInt();
        int n3 = sc.nextInt();
        if (n1 % 2 == 0 || n2 % 2 == 0 || n3 % 2 == 0) {
            System.out.println("Hay un número par");
        } else {
            System.out.println("No hay números pares");
        }
    }
}

File: PYTHON/Ejercicio4.py
'''
Escribe un programa que solicite al usuario ingresar tres números enteros. El programa
deberá imprimir en pantalla el mensaje "Hay un número par" si al menos uno de los números
ingresados es par. De lo contrario, deberá imprimir en pantalla el mensaje "No hay
números pares".
'''
n=input("Introduce tres numeros enteros separados por espacios: ")
n1, n2, n3 = map(int, n.split())
if n1%2==0 or n2%2==0 or n3%2==0:
    print("Hay un número par")
else:
    print("No hay un números pares")